wfl/fit/error.py

```diff
@@ -379,10 +379,16 @@ def select_units(prop, plt_type):
         prop = re.sub(r"/comp\b", "", prop)
     if "forces" in prop:
         prop = "forces"
-    try:
-        return units_dict[prop][plt_type]
-    except KeyError:
-        return "unknown units"
+
+    if "energy/atom" in prop:
```

Comment:
What's the specific use case for allowing arbitrary properties that happen to include things like `"energy/atom"` ?

Reply:
Is it other `"/<something"` suffixes?

Reply:
If so, might be nice to put in a comment to explain that.

Reply:
I'm not sure it's a good idea to have things that could match such arbitrary property names.  Is it too complicated to only guess the exact names of a few specific properties, and pass a dict of units (property names are keys) that can extend/override the built-in guesses?

Reply:
Ok, let me add this.